api/term/service.py

```diff
@@ -3,7 +3,7 @@
 
 
 def get_all_terms(page, page_size, order_by):
-    return [t.as_dict() for t in sort_and_paginate(Term.query, order_by, page, page_size).all()]
+    return (t.as_dict() for t in sort_and_paginate(Term.query, order_by, page, page_size).all())
```

Comment:
https://stackoverflow.com/questions/43808180/python-3-x-list-comprehension-vs-tuple-generator
Switch back to list comprehension

Reply:
Why?